src/editors/playgroundController.ts

```diff
@@ -76,6 +85,34 @@ export default class PlaygroundController {
         log.info('Active editor path', editor.document.uri?.path);
       }
     });
+
+    vscode.window.onDidChangeTextEditorSelection((editor) => {
+      this._selection = editor.selections
```

Comment:
Are the selections always in order here? If a user selects a few lines lower in a file and then `alt+click` selects a few more lines higher up in the code are they in chronological order?

Reply:
Very nice catch! 👏